packages/app/src/components/learningPaths/LearningPathsPage.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { SearchContextProvider } from '@backstage/plugin-search-react';
+import { Content, Page, InfoCard, Header } from '@backstage/core-components';
+import {
+  CircularProgress,
+  Grid,
+  Link,
+  makeStyles,
+  Typography,
+} from '@material-ui/core';
+
+import useSWR from 'swr';
+import { fetcher, ErrorReport } from '../../common';
+
+const useCatalogStyles = makeStyles({
+  root: {
+    height: '100%',
+    transition: 'all .25s linear',
+    textAlign: 'left',
+    '&:hover': {
+      boxShadow: '0px 0px 16px 0px rgba(0,0,0,0.8)',
+    },
+    '& svg': {
+      fontSize: 80,
+    },
+  },
+  subheader: {
+    display: 'block',
+    width: '100%',
+  },
+  link: {
+    '&:hover': {
+      textDecoration: 'none',
+    },
+  },
+});
+
+const LearningPathCards = () => {
+  const classes = useCatalogStyles();
+
+  const { data, error, isLoading } = useSWR('/homepage/data.json', fetcher);
```

Comment:
You can create a new /learning-path/data.json.
In the future the datasource for the homepage and learning path will be totally different.

Reply:
Resolved in the latest commit.